src/proposals/20240627/SparkGnosis_20240627.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { IERC20 }        from 'lib/erc20-helpers/src/interfaces/IERC20.sol';
+import { IExecutorBase } from 'lib/spark-gov-relay/src/interfaces/IExecutorBase.sol';
```

Comment:
I'm not sure I understand where it's used and why it's needed. Is it a part of future commits?